tutorial-getting-started/index.html

```diff
@@ -102,7 +102,7 @@ <h3 is-upgraded><strong>Access and Login to ChaosCenter Dashboard</strong></h3>
 <p>You have been assigned a default project with Owner permissions </p>
 <p class="image-container"><img style="width: 624.00px" src="img/home.png"></p>
 <h3 is-upgraded><strong>Verify Successful Registration of the Self Agent </strong></h3>
-<p>Once the project is created, the cluster is automagically registered as a chaos target via installation of Litmus Chaos Agents. This is represented as &#34;Self-Agent&#34; in the Agents console of the Chaos Center Dashboard. </p>
+<p>Once the project is created, the cluster is automatically registered as a chaos target via installation of Litmus Chaos Agents. This is represented as &#34;Self-Agent&#34; in the Agents console of the ChaosCenter Dashboard. </p>
```

Comment:
apparently, automagically is a thing in the infra world!

Reply:
> apparently, automagically is a thing in the infra world!

Okay, I didn't know about this. So shall we let it be?

Reply:
yes, we can